src/utils/validations.ts

```diff
@@ -125,11 +128,11 @@ export function validateChannelColor(color: string): void {
   }
 }
 
-export function validateChannelName(name: string) {
+export function validateChannelName(name: string, optionName: string) {
   debug('validateChannelName()')
   if (name.trim().length === 0) {
     throw (eInvalidValue("Channel name can't be empty or consist only of whitespace.", [
-      'Supply a non-empty value to --name flag',
+      `Supply a non-empty value to ${optionName} flag`,
```

Comment:
Change to:
`Supply a non-empty value to flag '${optionName}`

to be consistent with the other validation messages (lines 45-86). I see other validation messages diverged from that. I will open a separate issue for that.